pylink/__main__.py

```diff
@@ -597,6 +597,11 @@ def main(args=None):
 
     try:
         args.command(args)
+    except AttributeError:
```

Comment:
Rather than catch `AttributeError` would it be enough to check the value of `args.command` or even the length of `args`?

Reply:
Either works, I initially had implemented `if not hasattr(args, 'command')` but thought this was simpler.

I can revise as needed.

Reply:
`hasattr` seems reasonable to me - @hkpeprah what do you think?

Reply:
👍  for `hasattr` if only to ensure we don't inadvertently catch unrelated `AttributeError`s.